tests/blueant/test_checks.py

```diff
@@ -138,3 +141,129 @@ def __init__(
     )
 
     assert result is expected_passed
+
+
+@pytest.mark.parametrize(
+    (
+        "yaml_exists",
+        "rule_threshold",
+        "time_frame_str",
+        "events",
+        "current_count",
+        "expected_passed",
+    ),
+    [
+        (
+            True,
+            5,
+            "7d",
+            [
+                {
+                    "timestamp": datetime(2025, 7, 26, 12, 0, tzinfo=UTC).timestamp(),
+                    "metadata": {"num_items": 10},
+                }
+            ],
+            14,
+            True,
+        ),
```

Comment:
Pytest allows to keep params and id at one place, which makes test data easier to maintain, imo. Could you adapt?

```suggestion
        pytest.param(
            True,
            5,
            "7d",
            [
                {
                    "timestamp": datetime(2025, 7, 26, 12, 0, tzinfo=UTC).timestamp(),
                    "metadata": {"num_items": 10},
                }
            ],
            14,
            True,
            id="passes_within_threshold",
        ),
```